### DISCUSSION TITLE
feapder二次开发

### DISCUSSION THREAD

**Discussion by pikadoramon** [Boris-code/feapder#229] [Ideas]
我已经调研大佬的代码许久，发现有些特性在生产环境有着不错的效果。本人之前也对feapder作二次开发应用在不同场景，由于和master分支差别较大。后续打算将二次开发版本发布在github上。所以先征求大佬你的同意，我这边再公布在github上 @Boris-code

**Comment by Boris-code** [comment]
你这边主要是改了什么呀，互相学习，可以发的

**Reply by pikadoramon** [reply]
例如关于settings的导入使用，以对象形式进行单例导入。类似django, scrapy的settings对象。还有一些生产环境的一些工具类，例如读取apollo配置，ck表配置和rocksdb缓存，be like这些

**Comment by Boris-code** [comment]
好的，到时候挂一个我的仓库链接就好，也算是帮我打广告了

**Reply by pikadoramon** [reply]
没问题
